test/lib/apis/galaxy.js

```diff
@@ -43,29 +43,35 @@ describe('APIs', () => {
       fetch.__Rewire__('request', request);
 
       return galaxy('foo/bar').then(({ body: { foo } }) => {
-        expect(foo).to.equal('bar');
+        expect(foo).toBe('bar');
       });
     });
 
     it('rejects request errors', () => {
       const request = sinon.stub().yields(new Error('bad'));
       fetch.__Rewire__('request', request);
 
-      return expect(galaxy('foo/bar')).to.be.rejectedWith('bad');
+      return galaxy('foo/bar').catch(e => {
+        expect(e.message).toEqual('bad');
+      });
```

Comment:
This is not the same test, as it would silently pass if the promise did not reject.